README.md

```diff
@@ -14,7 +14,7 @@ I started this project to learn about reverse engineering big Javascript codebas
 
 If you benefit from this project, you can buy me a beer :beers: :+1:
 
-[![paypal](https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif)](https://www.paypal.me/timklingeleers)
+[![paypal](https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif)](https://www.paypal.me/eimarfandino)
```

Comment:
Did you intend to leave this in? Maybe acceptable own fork but not merging into somebody else's repo?